climada/hazard/tc_tracks_synth.py

```diff
@@ -166,6 +206,170 @@ def _one_rnd_walk(track, ens_size, ens_amp0, ens_amp, max_angle, rnd_vec):
     return ens_track
 
 
+def _random_uniform_ac(n_ts, autocorr, time_step_h):
+    """Generate a series of autocorrelated uniformly distributed random numbers.
+
+    This implements the algorithm described here to derive a uniformly distributed
+    series with specified autocorrelation (here at a lag of 1 hour):
+    https://stats.stackexchange.com/questions/48086/
+        algorithm-to-produce-autocorrelated-uniformly-distributed-number
+    Autocorrelation is specified at a lag of 1 hour. To get a time series at a
+    different temporal resolution (time_step_h), an hourly time series is generated
+    and resampled (using linear interpolation) to the target resolution.
+
+    Parameters:
+        n_ts (int): length of the series
+        autocorr: autocorrelation (between -1 and 1)
+
+    Returns:
+        numpy.ndarray of length n
+    """
```

Comment:
According to CLIMADA's coding conventions, docstrings should follow the NumPy style. From one of the developer meetings, I remember that, while older code in CLIMADA is still allowed to contain google-style docstrings, new code is required to follow the new convention. Old docstrings should be converted to the new convention whenever they are touched.

Reply:
Ok, I've converted to numpy style in commit e837bc88958cd756867fef5a7ef8d79571c8981c.